core/description.hpp

```diff
@@ -57,8 +70,8 @@
 	};
 #endif
 
-#ifdef description_XEH_FiredMan_CAManBase
-	class GVAR(ShotCount) {
-		firedMan = QUOTE(_this call FUNC(SC_FiredEH));
-	};
-#endif
+//#ifdef description_XEH_FiredMan_CAManBase
```

Comment:
I'm curious what the reason is for this commented code going into the repo is?

Reply:
The direct XEH firedMan EH has been replaced with ace eventhandlers - they parse the input data with their own values in a way to detect ace thrown grenades, explosives, etc.

Reply:
This entry is gone, all the eventhandlers are in PreInit